app/routes/run/view.js

```diff
@@ -3,14 +3,15 @@ import { inject as service } from '@ember/service';
 import RSVP from 'rsvp';
 
 export default AuthenticateRoute.extend({
-  queryParams: {auth: false},
   internalState: service(),
-  apiCall: service('api-call'),
+  queryParams: {auth: false},
+  store: service(),
+  
   model(params) {
     this._super(...arguments);
-    this.get('internalState').set('currentInstanceId', params.tale_id);
-    //this.set('queryParams', params);
-    return this.apiCall.fetchTaleData(params.tale_id);
```

Comment:
I'm actually not sure what the original intent was here, but the Tale model obtained from the store has all of the information that `fetchTaleData` appends.

Reply:
Interesting.... this used to all be manually done in the `controllers/run/view.js` file. On navigating to the view, the Tale would be fetched and we would need further context for display - things like attaching the creator and the instances and other associated records.

I refactored the huge block (previously known as `grabData`) into a new reusable function called `fetchTaleData` in `api-call.js`. It is possible that this logic was redundant even before I had refactored it, further emphasizing our need for a clean slate in terms of technical debt.

If things are still working as expected and displaying without error, then maybe this is no longer needed...

Reply:
Ah that code might have pre-dated some of the attribute changes on the backend Tale model. I believe that, currently, _every_ Tale will have these attributes and there isn't a need to attach them anymore.

Reply:
I am still a bit wary of removing the `fetchTaleData` stuff right now, as it is unclear if it is in fact still needed and/or if removing it would have any negative side-effects.

In fact, I see one right now - there is no Creator listed under my newly-created Tale:
![Screen Shot 2019-09-18 at 2 54 18 PM](https://user-images.githubusercontent.com/1413653/65181251-4272ac00-da24-11e9-993d-644493863637.png)

Compared with the same test on dashboard.stage.wholetale.org:
![Screen Shot 2019-09-18 at 2 57 08 PM](https://user-images.githubusercontent.com/1413653/65181434-a6957000-da24-11e9-92fe-19cd19154cf4.png)

Without fetching the `creator`, we won't have anything to display beneath the Tale's title. Similarly, we need to fetch the `image` for the metadata page to properly display attributes about the Tale's environment.